### PR TITLE
01 Chapter Presentation Update

### PR DESCRIPTION
**Pull request by AngelFelizR**

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -22,16 +22,107 @@ This website is being developed by the [Data Science Learning Community](https:/
...
 
+
 ## Book club meetings {-}
 
-- Each week, a volunteer will present a chapter from the book (or part of a chapter).
-  - **This is the best way to learn the material.**
-- Presentations will usually consist of a review of the material, a discussion, and/or a demonstration of the principles presented in that chapter.
-- More information about how to present is available in the [github repo](https://github.com/r4ds/bookclub-shinyui).
-- Presentations will be recorded, and will be available on the [Data Science Learning Community YouTube Channel](https://dslc.io/youtube).
+- Each week, a volunteer will present a chapter from the book after assigning it to themselves in the [Volunteers Spreadsheet](https://docs.google.com/spreadsheets/d/1HhA4HMB5LgpyPaF2KZbwwcdtB5RWfZrBbpJ2yXO0E-k/edit?usp=sharing)
```
Comment (lgibson7) [inline index.Rmd]: The shared notes are meant to be used by multiple book club cohorts, so it's best not to include links for a specific cohort unless explicitly stated. Please indicate that is the volunteer spreadsheet for bookclub-shinyui cohort 2.

File: index.Rmd [hunk 2]
```diff
@@ -22,16 +22,107 @@ This website is being developed by the [Data Science Learning Community](https:/
...
+-  Presentations: 
+   -  Review of material
+   -  Questions you have
+   -  Maybe live demo
+   
+<br>
+
+- More information about how to present is available at [github.com/r4ds/bookclub-shinyui](https://github.com/r4ds/bookclub-shinyui).
+
+- Presentations will be recorded, and will be available on the [Data Science Learning Community YouTube Channel](https://www.youtube.com/playlist?list=PL3x6DOfs2NGhoKz3RSvwGxJYoCvE4MbnM).
```
Comment (lgibson7) [inline index.Rmd]: The shared notes are meant to be used by multiple book club cohorts, so it's best not to include links for a specific cohort unless explicitly stated. Please link to the DSLC YouTube channel as <https://dslc.io/youtube>, and you can separately include a link to the playlist for bookclub-shinyui cohort 2 if you'd like. 

Note: The meeting recordings and chat log for each chapter are appended to the notes, so adding the playlist may be unnecessary. (Ex. bookclub-shinyui chapter 1 [meeting recording](https://r4ds.github.io/bookclub-shinyui/meeting-videos.html))

**Review by lgibson7 — CHANGES_REQUESTED** [review]

File: index.Rmd (hunk 1 above)
Comment (AngelFelizR) [reply index.Rmd]: I understand, it's better to remove the link.

File: index.Rmd (hunk 2 above)
Comment (AngelFelizR) [reply index.Rmd]: No problem

**Review by lgibson7 — APPROVED** [review]
Looks good. Thanks Angel!

**Pull request merged**